node/node.go

```diff
@@ -748,6 +748,8 @@ func getRouterConfig(conf *config.Config, appClient abciclient.Client) p2p.Route
 		}
 
 	}
+	opts.HandshakeTimeout = 15 * time.Second
```

Comment:
Are these "small" values? They seem rather long, relative to other p2p timeouts (e.g., the gap between dial attempts, for example).

No objection to the values, I just wasn't clear if we have a rubric or are just setting safety margins. (The latter would also be fine)

Reply:
Yeah, that's true, these values are not particularly small. Safety margins is a better description for what they are, ensuring that we don't try to handshake a peer indefinitely.